docs/reference/configuration/loadpoints.md

```diff
@@ -347,6 +347,9 @@ Definiert die Anzahl der Phasen mit welcher die Wallbox angeschlossen ist.
 ```yaml
   phases: 1
 ```
+Hinweis: Der Wert bei `phases` ist massgeblich verantwortlich für die Berechnung der Ein-/Abschaltschwelle. Ist die Wallbox zwar 3-phasig angeschlossen, soll aber aufgrund geringer PV-Gesamtleistung im Überschussbetrieb nur 1-phasig betrieben werden (2 Phasen werden "abgeschaltet") dann ist hier ebenfalls der Wert `1` empfehlenswert.
```

Comment:
@VolkerK62 das ist doch Quatsch, wie kommst Du da drauf? Eine nicht schaltbare Box lädt immer so wie sie elektrisch angeschlossen ist?!

Reply:
Evtl hab ich mich blöd ausgedrückt. 
Das ist für die Fälle gedacht, wo z.B. mittels Lasttrennschalter für das Überschussladen 2 Phasen weggeschaltet werden.
Mancher wundert sich dann, warum nicht geladen wird, obwohl mehr als 1,4 kW Überschuss vorhanden ist. So wie ich es verstanden habe, ist einer der Gründe, dass `phases: 3` in der yaml eingetragen ist und somit grob 4 kW Überschuss vorhanden sein müssen, damit evcc das Laden einschaltet. 
Darauf wollte ich hinweisen. Besonders für Neueinsteiger. Wenn man sich nach einiger Zeit ein wenig auskennt, ist der Hinweis eigentlich nicht nötig.
Evtl wäre so ein Hinweis auch besser in den FAQ aufgehoben? 
Wenn ich die Bedeutung von `phases` allerdings falsch verstanden habe, dann schmeiß den PR einfach weg

Reply:
Ich denke der Hinweis muss v.a. Klar sein. Relevant sind die effektiven Phasen- die entstehen aus Fahrzeug und WB. Evcc kann die entsprechenden Hinweise aufnehmen. Wenn an einer 3p Box auch nur entfernt die Chance besteht, dass ein 3p Fzg geladen werden könnte dann führt jede Konfiguration als 1p einfach zu Fehlern. mit Lasttrennschalter ist die Box ja physisch als 1p angeschlossen, dann passt es. 

Long story short: die Beschreibung muss halt sachrichtig sein.

Reply:
Da hast du natürlich völlig recht. Schlecht gemacht. Wie kann ich den PR zurückziehen? (close pull request?)

Als Neueinsteiger war ich erstmal an vielen Stellen in der Doku völlig lost. 
Gut wäre eine Beschreibung der internen Berechnung und welchen Einfluss die verschiedenen Parameter haben. Praktisch ein "evcc für Dummies" :-)  Aber das ist bestimmt ein Haufen Arbeit und ich habe ehrlich gesagt keine Idee, wie man so etwas am Besten aufsetzt. Siehe oben, Beschreibungen verfassen ist nicht meine Kernkompetenz ;-)

Reply:
Meine leider auch nicht. Da sind wir für Hilfe dankbar, es muss nur inhaltlich richtig sein ;)